app/Http/Controllers/AdminController.php

```diff
@@ -7,29 +7,108 @@
 use App\Models\User;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Session;
 
 class AdminController extends Controller
 {
     /**
      * Display a listing of the resource.
      */
-    public function index()
-    {
-       return view('Admin.user.admin.allAdmin');
+    // public function index(Request $request)
+    // {
+
+   
+    // $slug = $request->get('slug');
+    // $email = $request->get('email');
+    // $active = $request->has('status') && $request->get('status') === 'active';
+    // $blocked = $request->has('status') && $request->get('status') === 'blocked';
+
+    // $query = Admin::query()
+    //     ->with('userAdmin')
+    //     ->leftJoin('users', 'users.id', '=', 'admins.user_id');
+
+    // if ($slug) {
+    //     $query->where('users.slug', $slug);
+    // }
+
+    // if ($email) {
+    //     $query->where('users.email', $email);
+    // }
+
+    // if ($active) {
+    //     $query->where('users.active', 1);
+    // }
+
+    // if ($blocked) {
+    //     $query->where('users.active', 0);
+    // }
+
+    // // $query->where('users.position', 'admin'); // Filter for admin position
+    // $admins = $query->select('admins.*')->paginate(25)->appends(['slug' => $slug, 'email' => $email]);
+
+        
+    //     return view('Admin.user.admin.allAdmin', compact('admins'));
+    // }
+
+    public function index(Request $request)
```

Comment:
I notice that you don't format your code properly, make sure to format it 

Here is a trick 
if you are using PHPStorm press `Ctrl`+ `Alt` + `L` from the keyboard

if you are using VSCode download the extension `phpfmt` when you want to format your code press `Alt` + `Shift` + `F`